stac_fastapi/pgstac/core.py

```diff
@@ -203,8 +204,8 @@ async def _add_item_links(
                 and all([collection_id, item_id])
             ):
                 feature["links"] = await ItemLinks(
-                    collection_id=collection_id,
-                    item_id=item_id,
+                    collection_id=collection_id,  # type: ignore
+                    item_id=item_id,  # type: ignore
```

Comment:
collection_id and item_id should always be define but for some reason mypy thinks it can be None